src/ngraph/runtime/cpu/pass/cpu_workspace_insertion.cpp

```diff
@@ -52,7 +52,9 @@
 #include "ngraph/runtime/cpu/op/max_pool_with_indices.hpp"
 #include "ngraph/runtime/cpu/op/sigmoid.hpp"
 
-void ngraph::runtime::cpu::pass::CPUWorkspaceInsertion::construct_max_pool_with_indices()
+using namespace ngraph;
+
+static std::shared_ptr<pattern::Matcher> create_maxpool_with_indices_matcher()
```

Comment:
"create_maxpool_backprop_pattern"

Reply:
it's less of a pattern more of a parameter object 😄  this is intentionally named differently from a fusion naming convention.